commit 35a4a1ed1f0fc6f1cbccd31bb7db9431b2518a5b
Author:     Andrew Bartlett <abartlet@samba.org>
AuthorDate: Fri Sep 14 11:58:02 2012 -0700
Commit:     Andrew Bartlett <abartlet@samba.org>
CommitDate: Fri Sep 14 22:53:30 2012 +0200

    build: Fix build on systems without ldap development headers
    
    Autobuild-User(master): Andrew Bartlett <abartlet@samba.org>
    Autobuild-Date(master): Fri Sep 14 22:53:30 CEST 2012 on sn-devel-104
---
 source3/wscript_build | 1 +
 1 file changed, 1 insertion(+)

diff --git a/source3/wscript_build b/source3/wscript_build
index 60965ef..7e39ec3 100755
--- a/source3/wscript_build
+++ b/source3/wscript_build
@@ -740,6 +740,7 @@ bld.SAMBA3_LIBRARY('smbldaphelper',
                    source='passdb/pdb_ldap_schema.c passdb/pdb_ldap_util.c',
                    deps='smbldap secrets3',
                    allow_undefined_symbols=True,
+                   enabled=bld.env.HAVE_LDAP,
                    private_library=True)
 
 bld.SAMBA3_SUBSYSTEM('SERVER_MUTEX',
